starboard/shared/uwp/extended_resources_manager.h

```diff
@@ -49,6 +49,7 @@ class ExtendedResourcesManager {
 
   // Returns true when the d3d12 device and command queue can be used.
```

Comment:
Add to this comment to include the buffer heap

Reply:
Done